### PR TITLE
Update auto-update config

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Currently, auto update is a bit noisy with all its messages in PRs. Experimenting with our GH auto-update config so hopefully its less noisy, particularly for long-running PRs

## Checklist

- [x] Needs review from frontend eng team

### CODE REVIEW

File: .github/workflows/autoupdate.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ on:
   # This will trigger on all pushes to all branches.
   push: {}
```
Comment (john-rock) [inline .github/workflows/autoupdate.yml]: We can't have two `push` commands let's comment or remove this one so it doesn't conflict with the line 7 `push`.

**Review by john-rock — CHANGES_REQUESTED** [review]
Line 5 just needs to be removed or commented and then I can approve!

File: .github/workflows/autoupdate.yml (hunk 1 above)
Comment (nghi-ly) [reply .github/workflows/autoupdate.yml]: oops, i missed that! great catch!

Comment (nghi-ly) [inline .github/workflows/autoupdate.yml]:
```suggestion
#  push: {}
```

**Comment by nghi-ly** [comment]
made the change, @john-rock . can i get a re-review, please? 🙏

**Review by john-rock — APPROVED** [review]
lgtm! Let's 🚢 and monitor the results to make sure we are seeing expected behavior and decreased noise.

**Pull request merged**